test/models/report_test.rb

```diff
@@ -3,7 +3,19 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test '#editable?' do
+    target_user = User.new
+    non_target_user = User.new
+    report = target_user.reports.new
+
+    assert report.editable?(target_user)
+    assert_not report.editable?(non_target_user)
```

Comment:
この程度の行数なら好みの世界ですが、僕は変数の寿命を短くするスタイルが好きですね。

```suggestion
    target_user = User.new
    report = target_user.reports.new
    assert report.editable?(target_user)

    non_target_user = User.new
    assert_not report.editable?(non_target_user)
```

Reply:
なるほど！
その方が読みやすくも感じますね。修正しました。